lib/ArrayMap.js

```diff
@@ -1,57 +0,0 @@
-/*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
-*/
-
-"use strict";
-
-class ArrayMap {
-	constructor() {
-		this.keys = [];
-		this.values = [];
-	}
-
-	get(key) {
-		for(let i = 0; i < this.keys.length; i++) {
-			if(this.keys[i] === key) {
-				return this.values[i];
-			}
-		}
-		return;
-	}
-
-	set(key, value) {
-		for(let i = 0; i < this.keys.length; i++) {
-			if(this.keys[i] === key) {
-				this.values[i] = value;
-				return this;
-			}
-		}
-		this.keys.push(key);
-		this.values.push(value);
-		return this;
-	}
-
-	remove(key) {
```

Comment:
Need to check if anything was calling `.remove`. The native `Map` prototype method is `delete`.